translations/actions/actions/pt-br.yaml

```
@@ -0,0 +1,5 @@
+title: Use @action, '{{on}}', e '{{fn}}' no lugar das actions
+description: >
+  Na classe JavaScript, use o decorator '<code>'@action'</code>' para sinalizar a função que você gostaria de chamar no template.
+  No template, use o modifier '<code>{{on}}</code>' para marcar quando chamar uma função.
+  Se precisa passar um argumento para a função, use o helper '<code>{{fn}}</code>' também. ('<a href="https://guides.emberjs.com/release/components/component-state-and-actions/#toc_html-modifiers-and-actions" target="_blank" rel="noopener noreferrer">'Veja o Ember Guides para saber mais'</a>'.)
```

Can we keep the line formatting from `en-us.yaml`?

For some context, we used to have 1 large translations file. It was later split into multiple files, one for each section.

I went further by giving each sentence its own line, in case this could help with future translations and updates to translations (possibly more visible code diff in GitHub).

```suggestion
  Se precisa passar um argumento para a função, use o helper '<code>{{fn}}</code>' também.
  ('<a href="https://guides.emberjs.com/release/components/component-state-and-actions/#toc_html-modifiers-and-actions" target="_blank" rel="noopener noreferrer">'Veja o Ember Guides para saber mais'</a>'.)
```